REFERENCE.md

```diff
@@ -180,9 +180,9 @@ Empirically, we've found orthogonalization to be essential in obtaining accurate
 
 In the sections above on `min.node.size`, `alpha`, and `imbalance.penalty`, we described how tree training protects against making splits that result in a large size imbalance between the children, or leaf nodes that are too small. In a causal setting, it is not sufficient to consider the number of examples in each node -- we must also take into account the number treatment vs. control examples. Without a reasonable balance of treated and control examples, there will not be enough information in the node to obtain a good estimate of treatment effect. In the worst case, we could end up with nodes composed entirely of control (or treatment) examples.
 
-For this reason, causal splitting uses modified notions of each split balancing parameter:
+For this reason, causal splitting (enabled with the argument `stabilize.splits = TRUE`) uses modified notions of each split balancing parameter:
```

Comment:
I find it a little confusing to say "enabled with the argument `stabilize.splits = TRUE`" because it is already enabled by default. We actually don't expect users to disable it, as it's proved important to performance, so I'm not sure we need to call out the argument at all.